src/shared/ui/components/BaseTable.vue

```diff
@@ -115,8 +114,8 @@ const { width } = useWindowSize();
 const PAGINATION_BREAKPOINT = 960;
 
 const items = computed(() => {
-  if (props.loading && props.pagination) {
-    return Array.from({ length: props.pagination.page_size }, () => null);
+  if (props.loading) {
+    return [null];
```

Comment:
Why this change at all?

Reply:
In order to get back to previous expression. It doesn't change anything since display is the same it was before.

Reply:
It used to be that if there is a pagination and no data yet, then it will display `page_size` placeholders. Now it ignores pagination and just displays a single placeholder. So, the display is not the same, or am I missing something?

Reply:
We have the following condition 
```
v-if="item && (width >= props.breakpoint || !$slots['mobile-card'])" 
```
so either it one or 10 length nullish array - anyway we will display only one row containing BaseLoading component.